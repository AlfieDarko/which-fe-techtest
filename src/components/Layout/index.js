import React from 'react'
import PropTypes from 'prop-types'

const Layout = ({ children }) => {
  return (
        <div className="layout" data-testid="layout">
            {children}
        </div>
  )
}

Layout.propTypes = {
  children: PropTypes.node
}

export default Layout
