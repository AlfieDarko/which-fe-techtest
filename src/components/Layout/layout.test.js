import React from 'react'
import { render } from '@testing-library/react'

import Layout from '.'

test('should render correctly', () => {
  const { asFragment } = render(<Layout />)
  expect(asFragment()).toMatchSnapshot()
})
