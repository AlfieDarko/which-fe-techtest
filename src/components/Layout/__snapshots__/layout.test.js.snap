// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<DocumentFragment>
  <div
    class="layout"
    data-testid="layout"
  />
</DocumentFragment>
`;
